manifests/a/acrosync/DuplicacyWeb/1.6.3/acrosync.DuplicacyWeb.locale.en-US.yaml

```
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: acrosync.DuplicacyWeb
+PackageVersion: 1.6.3
+PackageLocale: en-US
+Publisher: Acrosync

```

```suggestion
Publisher: Acrosync LLC
```
This should match control panel